src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Tests/Fixtures/FilterMetaGeneratorExtensions.cs

```diff
@@ -0,0 +1,62 @@
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Fixtures;
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Tests.Fixtures;
+
+public static class FilterMetaGeneratorExtensions
+{
+    public static Generator<FilterMeta> DefaultFilterMeta(this DataFixture fixture)
+        => fixture.Generator<FilterMeta>().WithDefaults();
+
+    public static Generator<FilterMeta> DefaultFilterMeta(this DataFixture fixture, int options)
+        => fixture.Generator<FilterMeta>()
+            .WithDefaults()
+            .WithOptions(fixture.DefaultFilterOptionMeta().GenerateList(options));
+
+    public static Generator<FilterMeta> WithDefaults(this Generator<FilterMeta> generator)
+        => generator.ForInstance(s => s.SetDefaults());
+
+    public static Generator<FilterMeta> WithIdentifier(this Generator<FilterMeta> generator, string identifier)
+        => generator.ForInstance(s => s.SetIdentifier(identifier));
+
+    public static Generator<FilterMeta> WithLabel(this Generator<FilterMeta> generator, string label)
+        => generator.ForInstance(s => s.SetLabel(label));
+
+    public static Generator<FilterMeta> WithHint(this Generator<FilterMeta> generator, string hint)
+        => generator.ForInstance(s => s.SetHint(hint));
+
+    public static Generator<FilterMeta> WithOptions(
+        this Generator<FilterMeta> generator,
+        IEnumerable<FilterOptionMeta> options)
+        => generator.ForInstance(s => s.SetOptions(options));
+
+    public static Generator<FilterMeta> WithOptions(
+        this Generator<FilterMeta> generator,
+        Func<IEnumerable<FilterOptionMeta>> options)
+        => generator.ForInstance(s => s.SetOptions(options));
+
+    public static InstanceSetters<FilterMeta> SetDefaults(this InstanceSetters<FilterMeta> setters)
+        => setters
+            .SetDefault(m => m.Identifier)
+            .SetDefault(m => m.Label)
+            .SetDefault(m => m.Hint)
+            .Set(m => m.Options, []);
```

Comment:
do we not want to set any default options?

Reply:
Think for this case, it's fine for it to be empty as it's the base `SetDefaults`. We need to be able to generate a default without its relationships populated as you're not necessarily going to need them in lots of tests.

However, for where we do want to use generated defaults, we'd use the `DataFixture.DefaultFilterMeta` extension instead e.g.

```cs
var filterMeta = DataFixture.DefaultFilterMeta(options: 5);
```

I've actually just removed the line you commented on and replaced it with a default on the `FilterMeta` entity itself

Reply:
cool